### DISCUSSION TITLE
Adding additional rulesets to so-octopus/Playbooks does not result in them being imported (2.4.20)

### DISCUSSION THREAD

**Discussion by thedeadliestcatch** [Security-Onion-Solutions/securityonion#11590] [2.4]
I have properly followed the instructions at https://docs.securityonion.net/en/2.4/playbook.html:

> Go to [Administration](https://docs.securityonion.net/en/2.4/administration.html#administration) –> Configuration.
>     At the top of the page, click the Options menu and enable the Show all configurable settings, including advanced settings. option.
>     On the left side, navigate to soctopus –> playbook –> rulesets.
>     On the right side, add one or more of the following: application,category,cloud,compliance,linux,macos,network,web,windows. These are based on the top level directories from the Sigma community repository rule’s folder.
>     At the top of the page, click the SYNCHRONIZE GRID button under the Options menu.

sooctopus is enabled and I have synchronized the grid manually.

The rulesets added included "linux" and others.

Unfortunately, none other than the default IDH and windows rulesets are to be found in Playbooks. The expected behavior is to see the new rules imported in Draft status.

**Comment by thedeadliestcatch** [comment]
I have re-opened this as an issue as it is clearly not an operator mistake.

**Comment by reyesj2** [comment]
Try this out

```
grep playbook_rulesets  /opt/so/conf/soctopus/SOCtopus.conf
```

Then let me know if the below command matches the output of the first command.
```
docker exec so-soctopus grep playbook_rulesets SOCtopus.conf
```

If they don't match run 
```
so-restart soctopus
```
Then
```
so-playbook-ruleupdate
```


Here is what my list of enabled playbook rulesets looks like in SOC
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94730068/ccd2c589-2f9f-4a64-af3c-3bfa8c274dfd)

Here is the output of so-playbook-ruleupdate running after I completed the above steps
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94730068/37047b9f-ca62-4fd4-bd51-21bfe6e68440)

**Comment by TOoSmOotH** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/11593
